src/backend/distributed/commands/index.c

```diff
@@ -411,14 +411,21 @@ PostprocessIndexStmt(Node *node, const char *queryString)
 		return NIL;
 	}
 
+	Oid schemaId = get_namespace_oid(indexStmt->relation->schemaname, true);
```

Comment:
worth adding a comment that we ensure schemaname is not NULL in PreprocessIndexStmt.